csss-site/src/events/frosh/templates/frosh/2023/frosh.html

```diff
@@ -0,0 +1,257 @@
+{% load staticfiles %}
+
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>CSSS RETRO FROSH 2023</title>
+
+		<!-- stylesheets -->
+		<link rel="stylesheet" href="{% static 'frosh_static/2023/css/root.css' %}">
+		<link rel="stylesheet" href="{% static 'frosh_static/2023/css/header.css' %}">
+		<link rel="stylesheet" href="{% static 'frosh_static/2023/css/flag.css' %}">
+		<link rel="stylesheet" href="{% static 'frosh_static/2023/css/button.css' %}">
+		<link rel="stylesheet" href="{% static 'frosh_static/2023/css/section.css' %}">
+
+		<!-- icon -->
+		<link rel="icon" type="image/png" href="{% static 'frosh_static/2023/images/icon.png' %}">
+
+		<!-- fonts from Google: IBM Plex Mono, Inter -->
+		<link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;700&family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+	</head>
+	<body>
+		<header>
+			<button onclick="location.href='https://sfucsss.org/'" class="icon">
+				<img src="{% static 'frosh_static/2023/images/CSSS.svg' %}" alt="CSSS' %}">
+			</button>
+			<div class="spacer"></div>
+			<a href="#frosh_week">
+				Frosh Week
+			</a>
+			<div class="vertical-bar"></div>
+			<a href="#schedule">
+				Schedule
+			</a>
+			<div class="vertical-bar"></div>
+			<a href="#sign_up">
+				Sign-Up
+			</a>
+			<div class="vertical-bar"></div>
+			<a href="#faq">FAQ</a>
+		</header>
+		<div class="header-spacer"></div>
+		<div class="flag flag-header">
+			<div class="background" style="background-image: url('{% static 'frosh_static/2023/images/flag_background.gif' %}');"></div>
+
+			<!-- thumbnails; smaller (<32kB) versions of the larger graphics;
+			     these are removed once the entire document is loaded fully. -->
+			<img onload="this.remove()" class="top-grid" src="{% static 'frosh_static/2023/images/flag_grid_t.png' %}" alt="">
+			<img onload="this.remove()" class="bottom-grid" src="{% static 'frosh_static/2023/images/flag_grid_t.png' %}" alt="">
+			<img onload="this.remove()" class="logo" src="{% static 'frosh_static/2023/images/flag_ht.png' %}" alt="CSSS RETRO FROSH 2023">
+
+			<img class="top-grid" src="{% static 'frosh_static/2023/images/flag_grid.png' %}" alt="">
+			<img class="bottom-grid" src="{% static 'frosh_static/2023/images/flag_grid.png' %}" alt="">
+			<img class="logo" src="{% static 'frosh_static/2023/images/flag_h.png' %}" alt="CSSS RETRO FROSH 2023">
+		</div>
+		<section>
+			<h1>Frosh Week</h1>
```

Comment:
w3c warning: 
> Consider using the h1 element as a top-level heading only (all h1 elements [are treated as top-level headings by many screen readers and other tools](https://html5accessibility.com/stuff/2021/07/08/preserved-in-html/)).

Change all h1s to h2 (and all associated styles, including h1 size), or make a custom header, `hc`, `h_custom`, `custom_h`, etc.


for reference, here are the defaults for h1:
```css
h1 {
    display: block;
    font-size: 2em;
    margin-before: 0.67em;
    margin-after: 0.67em;
    margin-start: 0;
    margin-end: 0;
    font-weight: bold; 
}
```